### DISCUSSION TITLE
Where to post leocad.dmp files?

### DISCUSSION THREAD

**Discussion by jackmojo1** [leozide/leocad#607] [General]
I'm getting a LOT of crashes since updating to 21.01.  Otherwise it's still the BEST Lego digital designing tool by FAR.  Anyway...

Per the crash dialog, it asks to submit the leocad.dmp file to you -- not certain where to send that as everywhere I try to post it says "we don't support that file type."

**Comment by jackmojo1** [comment]
Also -- I've gotten SO much use out of LeoCad that I'd really love to donate some dough your way.  How might I do that?

**Comment by leozide** [comment]
You have to zip or rename (change the extension to .log for example) the dump file before uploading. Is there a particular action you're doing that crashes the app?

I'm happy to know that you enjoy using the app. I didn't write it for money but I'd consider a donation to your favorite charity the same as donating to me.

**Reply by jackmojo1** [reply]
Duh -- Zip file.  Sorry, don't why I didn't think of that.

Also, the chief action I'm doing is dragging a color from the list over to a piece.  It's happened others too but SO many crashes if I click/drag a color.

I've worked around most of the crashes by using the "paint" tool to recolor pieces.
[leocad.zip](https://github.com/leozide/leocad/files/5859424/leocad.zip)

**Comment by leozide** [comment]
I looked at the crash, it's something inside your video card driver (which seems to be 2 years old). Have you tried updating it?
